ar4_hardware_interface/urdf/ar4.urdf.xacro

```diff
@@ -3,7 +3,7 @@
   <xacro:arg name="serial_port" default="/dev/ttyACM0"/>
   <xacro:arg name="arduino_serial_port" default="/dev/ttyUSB0"/>
   <xacro:arg name="calibrate" default="True"/>
-  <xacro:arg name="include_gripper" default="false"/>
+  <xacro:arg name="include_gripper" default="True"/>
```

Comment:
We probably want to have the gripper always included, right?

Reply:
Yes, Taking into consideration that we are on Gripper branch.